server/metrics/metrics.go

```diff
@@ -17,22 +17,29 @@ const (
 )
 
 type Metrics interface {
+	// Histogram metrics
```

Comment:
Can you elaborate on the purpose of these comments? At first glance, this isn't needful information for the client. Furthermore, grouping them by type isn't meaningful -- if we were going to reorganize, I'd propose grouping them by subsystem.